src/ui/templates/index.njk

```diff
@@ -16,17 +16,17 @@
     <link rel="apple-touch-icon" href="/assets/images/govuk-apple-touch-icon.png">
 
     <link rel="stylesheet" href="/assets/css/styles.css" />
-    <!--[if IE 8]>
-      <link rel="stylesheet" href="/govuk-frontend-ie8-3.5.0.min.css">
-    <![endif]-->
 
+    <meta property="og:locale" content="en_GB" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="" />
+    <meta property="og:description" content="" />
     <meta property="og:image" content="/assets/images/govuk-opengraph-image.png">
+    <meta property="og:image:width" content="512" />
+    <meta property="og:image:height" content="512" />
   </head>
 
   <body class="govuk-template__body govuk-body">
-
-    <script src="/assets/js/jsEnabled.js" type="text/javascript"></script>
```

Comment:
jsEnabled removed - I don't think we need this actually. Looks like the script/map still exists though, can probably remove now?